### PR TITLE
Add an optional parameter to /window call

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Addresses https://github.com/appium/appium/issues/10319

### CODE REVIEW

File: lib/protocol/routes.js [hunk 1]
```diff
@@ -103,7 +103,10 @@ const METHOD_MAP = {
   },
   '/wd/hub/session/:sessionId/window': {
     GET: {command: 'getWindowHandle'},
-    POST: {command: 'setWindow', payloadParams: {required: ['name']}},
+    POST: {command: 'setWindow', payloadParams: {
```
Comment (jlipps) [inline lib/protocol/routes.js]: really what we mean is that either name or handle may be present, but at least one _must_ be. is there a way to express that here?

Comment (imurchie) [reply lib/protocol/routes.js]: Where is the `name` coming from? The spec seems to just want a mandatory `handle`.

Comment (mykola-mokhnach) [reply lib/protocol/routes.js]: I'm not sure where the `name` comes from. It just was there. I could change the current implementation and add a validation helper, which is gonna check if either parameter is set

Comment (jlipps) [reply lib/protocol/routes.js]: `name` is [JSONWP](https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol#sessionsessionidwindow), `handle` is W3C

Comment (jlipps) [reply lib/protocol/routes.js]: this is basically the same situation as sendKeys

Comment (mykola-mokhnach) [reply lib/protocol/routes.js]: ok, I've added makeArgs call similar to sendKeys

**Pull request merged**